src/main/java/racingcar/model/RacingCar.java

```diff
@@ -0,0 +1,65 @@
+package racingcar.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import racingcar.exception.CarNameEmptyException;
+import racingcar.exception.CarNameOverMaxLengthException;
+
+public class RacingCar {
+
+    public static final int MOVE_FORWARD_THRESHOLD = 4;
+
+    private String name;
```

Comment:
```suggestion
    private final String name;
```
자동차의 이름이 정해지면 바뀔 일이 없으니 final로 선언해보는 것은 어떨까요?

Reply:
사실 이부분을 게임이라고 생각하여 이름 변경이 있을 수 있다고 생각해서 final로 설정하지 않았습니다.